app/hub/Views/CreateAccountView/CreateAccountViewContainer.jsx

```diff
@@ -219,7 +207,6 @@ CreateAccountViewContainer.propTypes = {
 		setToast: PropTypes.func.isRequired,
 		register: PropTypes.func.isRequired,
 		getUser: PropTypes.func.isRequired,
-		updateAccountPromotions: PropTypes.func.isRequired,
```

Comment:
I believe this should be replaced with the function you renamed `optIntoPromotions`

Reply:
Pesky proptypes at it again. Good catch!